legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/ingestmode/UnitemporalSnapshotAbstract.java

```diff
@@ -59,6 +59,18 @@ default boolean partitioned()
         return !partitionFields().isEmpty();
     }
 
+    @Value.Default
```

Comment:
Not critical but we may consider putting all the partition-related config in a class, since we have 6 parameters now

Reply:
Yeah let's do in the next iteration